test/fixtures/templates/good/resources/sqs/retention_period.yaml

```diff
@@ -0,0 +1,8 @@
+---
+AWSTemplateFormatVersion: "2010-09-09"
+Resources: 
+  MyQueue: 
+    Type: AWS::SQS::Queue
+    Properties: 
+      QueueName: "SampleQueue"
+      MessageRetentionPeriod: 5
```

Comment:
Thanks for adding a valid template as well, [looks like this picked up a false positive](https://travis-ci.com/github/aws-cloudformation/cfn-python-lint/jobs/367200676#L354)